src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.AddSub.cs

```diff
@@ -5,6 +5,68 @@ namespace System.Numerics
 {
     internal static partial class BigIntegerCalculator
     {
+        public static uint[] Add(uint left, uint right)
+        {
+            // Executes the trivial addition for two 32-bit integers.
+            // Since the caller is working with uint[] objects, we're
+            // doing him a favor with already providing it that way.
+
+            uint[] bits = new uint[2];
+
+            long digits = (long)left + right;
+            bits[0] = (uint)digits;
+            bits[1] = (uint)(digits >> 32);
+
+            return bits;
+        }
+
+        public static uint[] Add(uint[] left, uint right)
+        {
+            Debug.Assert(left != null);
+            Debug.Assert(left.Length >= 1);
+
+            // Executes the addition for one big and one 32-bit integer.
+            // Thus, we've just rather simplified code than below.
```

Comment:
Ah, I see.  I would instead say basically what you just explained to me, e.g. "Executes the addition for one big and one 32-bit integer.  No loop is necessary for the processing of 'right', as it's a single element, whereas 'left' is a set of elements."  Or something like that.